### PR TITLE
Fix repository release count when converting for API

### LINKED ISSUE #10946: [BUG] [API] RepoInfo incorect Release Counter
Gitea Version: 1.12.0+dev-72-g2c25e75dc

web show 18 releases: https://gitea.com/gitnex/GitNex/releases
api 0: https://gitea.com/api/v1/repos/Gitnex/gitnex

### PR DESCRIPTION
**Pull request by 6543**
fix #10946

### CODE REVIEW

File: models/repo.go [hunk 1]
```diff
@@ -353,6 +353,8 @@ func (repo *Repository) innerAPIFormat(e Engine, mode AccessMode, isParent bool)
 
 	repo.mustOwner(e)
 
+	numReleases, _ := GetReleaseCountByRepoID(repo.ID, FindReleasesOptions{IncludeDrafts: false, IncludeTags: true})
```
Comment (6543) [inline models/repo.go:355]: same way as UI calc it

Comment (lunny) [reply models/repo.go:355]: Then should we remove repo.NumReleases ?

Comment (6543) [reply models/repo.go:355]: @lunny your commend was hard to find :sweat_smile:  ... (commented on a "resolved" code comment)

done

**Comment by 6543** [comment]
@lunny I removed the **NumReleases** from the Repo struct

**Review by zeripath — APPROVED** [review]

**Review by mrsdizzie — APPROVED** [review]

**Comment by zeripath** [comment]
make lg-tm work

**Pull request merged**